basilisk/tests/test_anthropic_engine.py

```
@@ -0,0 +1,890 @@
+    def test_init_with_minimal_config(self, minimal_config, mock_anthropic_client):
+        """Test initialization with minimal configuration."""
+        engine = AnthropicEngine(minimal_config)
+
+        assert engine.config == minimal_config
+        assert engine.model == "claude-3-sonnet-20240229"  # default model
+        assert engine.client is not None
+
+    def test_init_sets_client_timeout(self, sample_config, mock_anthropic_client):
+        """Test that client timeout is properly set."""
+        with patch('basilisk.provider_engine.anthropic_engine.anthropic.AsyncAnthropic') as mock_class:
+            engine = AnthropicEngine(sample_config)
```

Ruff (0.11.9)

Local variable `engine` is assigned to but never used

Remove assignment to unused variable `engine`

(F841)